AFQ/api/bundle_dict.py

```diff
@@ -287,7 +287,7 @@ def baby_bd():
         "ATR_L", "ATR_R", "CST_L", "CST_R", "CGC_L", "CGC_R",
         "CCH_L", "CCH_R", "FP", "FA", "IFO_L", "IFO_R", "ILF_L",
         "ILF_R", "SLF_L", "SLF_R", "UNC_L", "UNC_R",
-        "ARC_L", "ARC_R", "MdLF_L", "MdLF_R"]
+        "ARC_L", "ARC_R"]
```

Comment:
@zikaste and @grotheer: can you confirm that we're dropping MdLF?

Reply:
Seems very likely given code further down: https://github.com/yeatmanlab/pyAFQ/pull/1071/files#diff-96764dfa545e2e28e90a01bb917265f5860ce2d39904199f2b41ca0f98455a6aR303. 

I think this means that we're still going to have MdLF bundles, but that they reuse the probability maps for ILF, for the first pass through.

Reply:
We are not exactly dropping MdLF, we are just using it as a "placeholder" for CCH because in the next step the probmap of MdLF is overwritten as the probmap of ILF anyways.